app/src/androidTest/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayerPluginTest.kt

```diff
@@ -266,11 +281,11 @@ class LocationLayerPluginTest {
 
         uiController.loopMainThreadForAtLeast(MAP_RENDER_DELAY)
 
-        val point: Point = mapboxMap.queryLocationSourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
+        val point: Point = mapboxMap.querySourceFeatures(LOCATION_SOURCE)[0].geometry() as Point
 
-        Assert.assertThat(plugin.locationEngine, nullValue())
-        Assert.assertEquals(point.latitude(), location.latitude, 0.1)
-        Assert.assertEquals(point.longitude(), location.longitude, 0.1)
+        assertThat(plugin.locationEngine, nullValue())
+        assertEquals(point.latitude(), location.latitude, 0.1)
```

Comment:
The selected delta atm: `The first decimal place is worth up to 11.1 km: it can distinguish the position of one large city from a neighboring large city`. I feel this can be made more accurate:

For example: `The third decimal place is worth up to 110 m: it can identify a large agricultural field or institutional campus.` or
`The fourth decimal place is worth up to 11 m: it can identify a parcel of land. It is comparable to the typical accuracy of an uncorrected GPS unit with no interference.`

https://gis.stackexchange.com/questions/8650/measuring-accuracy-of-latitude-and-longitude

Reply:
My testing revealed, that when zoomed out to approximately `0` the precision can be around two decimal points. For the test not to be flaky, I'm just checking if the point is in a general position of the forced location, as they are mostly being moved from the infinity anyway, leaving the responsibility for precise rendering to the Maps SDK.